resources/windows/windows.pkr.hcl

```diff
@@ -35,17 +56,24 @@ source "amazon-ebs" "win-source" {
   winrm_username = local.win_factory.user
   winrm_insecure = true
   winrm_use_ssl  = true
+  // By default Packer creates a default security group of [0.0.0.0/0] to access the instance
+  // Setting this value to true ensures only the public IP of the host can access the instance
+  temporary_security_group_source_public_ip = true
   launch_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/sda1"
     volume_size           = 60
     volume_type           = "gp2"
   }
+  run_volume_tags = {
```

Comment:
Just adding a few extra tags so that the build instance and volume match the tags of the finished AMI